webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetailsFile.scala

```diff
@@ -360,50 +376,6 @@ object UserFileProcessing {
     }
   }
 
-  def parseCaseSensitivityOpt(resource: UserFile): Either[UserConfigFileError, Boolean] = {
```

Comment:
This code was an almost direct duplication of the code above it. We don't do that for function that are that long (ie longer than 2 or 3 lines), especially when most of the code is just general purpose xml parsing code. This must be a red flag, the moment we know we are doing something that should not be done that way, and the time to think how to refactor a bit to avoid that consequence.